src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
Let's fix problem with eslint https://github.com/mate-academy/js_herbivores_and_carnivores/issues/952 

You can add static property after class declaration
```
class ClassName {
  ...
}

ClassName.staticProperty = value;
```